tests/state_test_util.go

```diff
@@ -146,7 +146,13 @@ func (t *StateTest) Run(subtest StateSubtest, vmconfig vm.Config) (*state.StateD
 	if logs := rlpHash(statedb.Logs()); logs != common.Hash(post.Logs) {
 		return statedb, fmt.Errorf("post state logs hash mismatch: got %x, want %x", logs, post.Logs)
 	}
-	root, _ := statedb.Commit(config.IsEIP158(block.Number()))
+	// Commit block
+	statedb.Commit(config.IsEIP158(block.Number()))
+	// Add 0-value mining reward. This only makes a difference in the cases
+	// where the coinbase suicided, or where the
```

Comment:
"or where the..." I think you missed the end of the sentence.

Reply:
Yeah... was going to look up if there were any other cases where it mattered, then forgot about it :)

Reply:
Ah, remembered now. So it also affects the cases where there are only 'bad' transactions, which aren't executed. In those cases, the coinbase gets no txfee, so isn't created. In those cases, this touch makes a difference .

Reply:
updated